Tests/PublicAPISpec.swift

```diff
@@ -114,6 +114,28 @@ class PublicAPISpec: QuickSpec {
                                                                pollInterval: .milliseconds(500))
             }
 
+            context("when calling closeMessage") {
+
+                it("will remove displayed campaign's view from hierarchy") {
+                    messageMixerService.mockedResponse = TestHelpers.MockResponse.withGeneratedCampaigns(
+                        count: 1, test: true, delay: 0,
+                        triggers: [Trigger(type: .event,
+                                           eventType: .loginSuccessful,
+                                           eventName: "e1",
+                                           attributes: [])])
+                    campaignsListManager.refreshList()
+                    RInAppMessaging.logEvent(LoginSuccessfulEvent())
```

Comment:
All the above setup could be refactored into a `displayLoginCampaign` helper function